source/ios/AdaptiveCards/ADCIOSFramework/ADCIOSFramework/ACRActionDelegate.h

```diff
@@ -0,0 +1,15 @@
+//
+//  ACRSubmitActionDelegate
+//  ACRSubmitActionDelegate.h
+//
+//  Copyright © 2017 Microsoft. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+@protocol ACRActionDelegate
+
+- (void)fetchUserResponses:(NSDictionary *)userResponses;
```

Comment:
>fetchUserResponses [](start = 8, length = 18)

This looks like a callback that the app receives when ACR has fetched user responses. If so, this method should be called `didFetchUserResponses`.

Likewise `didFetchHttpRequest`.